doc/release-notes.md

```diff
@@ -106,11 +106,22 @@ Low-level changes
 Configuration
 ------------
 
-* An error is issued where previously a warning was issued when a setting in
+- An error is issued where previously a warning was issued when a setting in
   the config file was specified in the default section, but not overridden for
   the selected network. This change takes only effect if the selected network
   is not mainnet.
 
+Network
+-------
+
+- When fetching a transaction announced by multiple peers, previous versions of
+  Bitcoin Core would sequentially attempt to download the the transaction from
```

Comment:
The first "the" in the "the the" before transaction could should be be dropped.

Reply:
Done